.gitignore

```diff
@@ -3,6 +3,7 @@ __pycache__/
 *.py[cod]
```

Comment:
similar to `.gitignore`, can you also add a `.dockerignore`? in addition to .idea, it will make sense to be sure and exclude e.g. `.venv` and other places a local virtualenv might lurk since those will be big and bloat the build context a lot